waving_hands/config.py

```diff
@@ -0,0 +1,33 @@
+import pathlib
+
+BASEPATH = pathlib.Path(__file__).parent
+DATA_PATH = BASEPATH / "data"
+
+LOGGING = {
+    "version": 1,
+    "formatters": {
+        "basic": {
+            "format": "[%(levelname)s] " "%(name)s::%(funcName)s() %(message)s"
+        }
+    },
+    "handlers": {
+        "console": {
+            "class": "logging.StreamHandler",
+            "level": "INFO",
+            "formatter": "basic",
+        }
+    },
+    "loggers": {
+        "waving_hands": {"level": "DEBUG", "handlers": ["console"]},
+    },
+}
+DATA = {
+    "spellbook": DATA_PATH / "spelllist.txt",
+    "names": {
+        "first": DATA_PATH / "groble_first_names.txt",
+        "last_first": DATA_PATH / "groble_last_first_compound.txt",
+        "last_second": DATA_PATH / "groble_last_second_compound.txt",
+        # Not currently used anywhere, and can be removed
+        "all": DATA_PATH / "groble_names.txt"
+    }
+}
```

Comment:
There are a bunch of different ways to setup logging: https://docs.python.org/3/library/logging.html I find dict logging the easiest, and cleaner than doing so in code.

Moving data into a folder was both for organization, and also allows `waving-hands` to be called from any directory on your filesystem once it's installed (the actual `config.py` full path is derived from `pathlib.Path(__file__)`, which can be used above to find the path to the other data files.